packages/wrangler/src/bundle.ts

```diff
@@ -340,7 +340,7 @@ export async function bundleWorker(
 		sourceRoot: destination,
 		minify,
 		metafile: true,
-		conditions: ["worker", "browser"],
+		conditions: ["workerd", "worker", "browser"],
```

Comment:
This indicates, I think, that our bundler will try workerd, then worker, then browser exports points before the standard node ones, right?